librz/core/cconfig.c

```diff
@@ -520,10 +422,13 @@ static void update_asmcpu_options(RzCore *core, RzConfigNode *node) {
 static bool cb_asmcpu(void *user, void *data) {
 	RzCore *core = (RzCore *)user;
 	RzConfigNode *node = (RzConfigNode *)data;
+	RzCmdStateOutput *state = NULL;
 	if (*node->value == '?') {
 		update_asmcpu_options(core, node);
 		/* print verbose help instead of plain option listing */
-		rz_asm_list(core, rz_config_get(core->config, "asm.arch"), node->value[1]);
+		state = malloc(sizeof(RzCmdStateOutput));
```

Comment:
this was not freed. In any case, I suggest you just allocate this on the stack:
```C
RzCmdStateOutput state = {0};
state.mode = RZ_OUTPUT_MODE_STANDARD;
rz_core_asm_plugins_print(core, rz_config_get(core->config, "asm.arch"), &state);
```

Reply:
omg you are right! Why did I even malloc?!

Reply:
I have implemented all of them, besides `mL`, which doesn't exist no more, and `LD`, which was never implemented.

Although, there are parts in the code where `rz_*_plugins_print` functions are not used, e.g. `aL`, `iL`, etc. That's because nothing is printed on screen when using, for example, `aLj`, even though `aL` and `aLq` work.